doc/dev-guide.md

```diff
@@ -507,7 +507,7 @@ Stripes modules of any type can provide a handler; the `handler` module type is
 
 Handlers are invoked in two situations: when core events occur, and when module-specified links are rendered:
 
-* These is a small, defined set of core events, which can be imported as `coreEvents` from `@folio/stripes-core/src/events`. This file exports an object with numeric constant members named `LOGIN`, `LOGOUT`, etc. When a relevant event occurs within Stripes, the handler is invoked with the relevant constant.
+* These are a small, defined set of core events, which can be imported as `coreEvents` from `@folio/stripes-core/src/events`. This file exports an object with numeric constant members named `LOGIN`, `LOGOUT`, etc. When a relevant event occurs within Stripes, the handler is invoked with the relevant constant.
```

Comment:
No: there _is_ a small _set_ (a singular noun). Alternatively, we might have written that there _are_ some core events (plural), but here you have introduced a number mismatch.

Reply:
Drat, i did not try out that combination. I will fix it.